deno.json

```diff
@@ -8,10 +8,10 @@
   },
   "fmt": { "exclude": ["LICENSE", "README.md", ".github/**/*.md"] },
   "tasks": {
-    "run": "deno run --env='.env' --allow-env='REPOSITORY_OWNER,REPOSITORY_NAME,REPOSITORY_PATH,GITHUB_TOKEN' --allow-net='0.0.0.0,api.github.com'",
+    "run": "deno run --env='.env' --allow-env --allow-net='0.0.0.0,api.github.com'",
```

Comment:
This is because Hono requires access to all environment variables. We can't specify them with arguments.
The change of `"test"` is the same reason.

```sh
┏ ⚠️  Deno requests env access.
┠─ Learn more at: https://docs.deno.com/go/--allow-env
┠─ Run again with --allow-env to bypass this prompt.
┗ Allow? [y/n/A] (y = yes, allow; n = no, deny; A = allow all env permissions) > 
```